importer/src/main/java/org/hiero/mirror/importer/parser/record/transactionhandler/EVMHookHandler.java

```diff
@@ -163,4 +191,48 @@ private HookType translateHookType(HookCase hookCase) {
             }
         };
     }
+
+    private void processMappingEntries(LambdaMappingEntries entries, long consensusTs, long ownerId, long hookId) {
+        final var mappingSlot = leftPadBytes(toBytes(entries.getMappingSlot()), 32);
+
+        for (final var entry : entries.getEntriesList()) {
+            final var mappingKey = entry.hasKey()
+                    ? leftPadBytes(toBytes(entry.getKey()), 32)
+                    : keccak256(toBytes(entry.getPreimage()));
+
+            final var valueWritten = toBytes(entry.getValue());
+            final var derivedSlot = keccak256(mappingKey, mappingSlot);
+            persistChange(ownerId, hookId, derivedSlot, valueWritten, consensusTs);
+        }
+    }
+
+    private void processStorageSlotUpdate(LambdaStorageSlot storageSlot, long consensusTs, long ownerId, long hookId) {
```

Comment:
Prefer not to shorten.

```suggestion
    private void processStorageSlotUpdate(LambdaStorageSlot storageSlot, long consensusTimestamp, long ownerId, long hookId) {
```